src/components/Dialogs/InstallFailed.vue

```diff
@@ -0,0 +1,55 @@
+<!--
+//  InstallFailed.vue
+//
+//  Created by Kalila L. on 22 Feb 2020.
+//  Copyright 2020 Project Athena and contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+<template>
+    <v-dialog
+        width="500"
+        persistent
+        v-model="showInstallFailed"
+    >
+        <v-card>
+            <v-card-title
+                class="headline"
+                primary-title
+                dark
+            >
+                <v-icon color="success" class="mr-2">mdi-message-alert</v-icon>
```

Comment:
```suggestion
                <v-icon color="success" class="mr-2">mdi-message-alert</v-icon>
```
                <v-icon color="red" class="mr-2">mdi-message-alert</v-icon>